frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import config from '../config';
+
+export async function pointsManipulation(teamID, action) {
+  try {
+    await axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`);
+  } catch(error) {
+    alert(error);
+  }
+}
```

Comment:
EOF someone has to install prettier

Reply:
Or run it after each commit? https://www.npmjs.com/package/husky

Reply:
That could be an excellent option, but in separated task/PR, for now run prettier to faster merge

Reply:
I expected to add husky with CI ;) to be honest